dnadoh/assemble.py

```diff
@@ -9,9 +9,18 @@
 
 def read_combined(stem):
     """Assemble data from a set of files."""
+    reference = read_reference_genome(stem)
     phenotypes = _read_phenotypes(stem)
+
     variants = _read_genomes(stem, phenotypes)
+    variants = variants.rename(columns={"base": "alternate"})
+    variants["reference"] = variants.apply(
+        lambda x: reference["genome"][x["loc"]-1], axis=1)
+    # re-arrange columns as convention is for reference to come before alternate
+    new_cols = ["loc", "reference", "alternate", "pid"]
+    variants = variants[new_cols]
     combined = phenotypes.set_index("pid").join(variants.set_index("pid"))
+    # combined = pd.merge(phenotypes, variants, on="pid", how="outer")
```

Comment:
suggestion for an alternative way of joining. i don't like the index column in pandas because i find that it hard to access the index column by name (maybe i just don't know enough about pandas)

Reply:
I really don't like Pandas' indexes either - yes, please make this change.